compiler/src/dotty/tools/dotc/fromtasty/Debug.scala

```diff
@@ -35,11 +35,11 @@ object Debug {
 
     val fromTastyOut = Files.createDirectory(tmpOut.resolve("from-tasty"))
 
-    val ext = "hasTasty"
-    val classes = Directory(fromSourcesOut).walk.filter(x => x.isFile && x.extension == ext).map { x =>
+    val extensions = List("tasty", "hasTasty")
+    val classes = Directory(fromSourcesOut).walk.filter(x => x.isFile && extensions.exists(_.toLowerCase == x.extension)).map { x =>
```

Comment:
Is extension supposed to always return lower cases? If not, we should do `x.extension.toLowerCase` to make sure it is as we expected.